Cython/Includes/cpython/context.pxd

```diff
@@ -0,0 +1,63 @@
+from .object cimport PyObject
```

Comment:
A file comment would be nice that states the minimum CPython version for this, just in case.

Reply:
Okay.